handbook/engineering/career-development/framework.md

```diff
@@ -2,10 +2,9 @@
 
 Our career development framework is here to help you understand the expectations of your role, and to provide a common language for you and your manager to discuss and plan your career growth. It is also an important part of our larger goal of ensuring everyone is equitably recognized for the impact they have at work, and to reduce bias in promotions and hiring.
 
-## What are these expectations?
+## What are the expectations of my role?
 
-The currently five levels for software engineers in our framework.
-A level is composed of three categories, each with a summary statement and several example behaviors. These categories are:
+There are currently five levels for software engineers described in our framework. A level is composed of three categories, each with a summary statement and several example behaviors. These categories are:
```

Comment:
Right now this page (here and in the lvl 6/7 cells) reads like "there are only 5 levels today but we plan to have 7 levels in the future", and I think it should read more like "there are 7 levels" (and it is just a ephemeral state that 2 of them are not documented yet).